### DISCUSSION TITLE
创建docker.sock

### DISCUSSION THREAD

**Discussion by xxtyyq** [jxxghp/MoviePilot#2427] [问题咨询]
已经创建moviepilot容器，还能增加docker.sock映射不？我看映射那里没有/var，我这方面真不太会，请教详细过程，感谢！

**Comment by cng10001** [comment]
你是什么环境


> 该配置无法在群晖或是其他厂商的官方ui界面配置，需要使用docker-compose或是ssh创建容器时才可以进行配置。
> 使用7.2以上的群晖系统或是portainer可以直接在ui界面上使用docker-compose(推荐)
> 极空间无法开启ssh，也无法使用该配置项

**Reply by xxtyyq** [reply]
群晖的docker，我忍不住重建了，已解决，多谢多谢

**Comment by zraxx** [comment]
可以用portainer 到容器内编辑，增加映射目录

**Reply by xxtyyq** [reply]
好的，谢谢！
